.github/workflows/x-keycloak-operator-hub-publish.yml

```diff
@@ -0,0 +1,155 @@
+        type: string
+      quay-org:
+        required: true
+        type: string
+      community-operators-repo:
+        required: true
+        type: string
+      prod-operators-repo:
+        required: true
+        type: string
+      mvn-url:
+        required: true
+        type: string
+      version:
+        required: true
+        type: string
+      tag:
+        required: true
+        type: string
+    secrets:
+      GH_TOKEN:
+        required: true
+      MVN_USERNAME:
+        required: true
+      MVN_TOKEN:
+        required: true
+      QUAY_USERNAME:
+        required: true
+      QUAY_TOKEN:
+        required: true
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Setup Java
+        uses: actions/setup-java@v3.0.0
+        with:
+          distribution: 'temurin'
+          java-version: 11
+
+      - name: Checkout Repository
+        uses: actions/checkout@v3.0.0
+        with:
+          repository: '${{ inputs.gh-org }}/keycloak'
+          token: ${{ secrets.GH_TOKEN }}
+          ref: ${{ inputs.tag }}
+          path: keycloak
+
+      - name: Login to Quay
+        uses: docker/login-action@v1.14.1
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_TOKEN }}
+
+      - name: Build
+        env:
+          MAVEN_ID: kc-rel-repository
+          MAVEN_URL: ${{ inputs.mvn-url }}
+          MAVEN_USERNAME: ${{ secrets.MVN_USERNAME }}
+          MAVEN_PASSWORD: ${{ secrets.MVN_TOKEN }}
+        working-directory: keycloak
+        # hardcoded Quay.io image coordinates here are used only for building the K8s YAML resources (where we use Quay and not Docker Hub)
+        run: |
+          ./mvnw clean package \
+              -s ./.github/mvn-rel-settings.xml \
+              -f operator/pom.xml \
+              -DskipTests \
+              -Dquarkus.container-image.image=quay.io/${{ inputs.quay-org }}/keycloak-operator:${{ inputs.tag }} \
+              -Dquarkus.container-image.build=true \
+              -Dquarkus.jib.image-id-file=image-id.txt \
+              -Doperator.keycloak.image=quay.io/${{ inputs.quay-org }}/keycloak:${{ inputs.tag }}
+
+      - name: Install Yq
+        working-directory: keycloak
+        run: sudo snap install yq
+
+      - name: Install OLM
+        working-directory: keycloak
+        run: ./operator/scripts/install-olm.sh
+
+      - name: Fetch previous version of Keycloak
+        working-directory: keycloak
+        # Could be used version of the last tag from the keycloak-bot/community-operators repo
+        run: echo ::set-output name=previous_tag::$(git describe --tags $(git rev-list --tags --max-count=1))
+
+      - name: Create OLM Bundle
+        working-directory: keycloak
+        run: ./operator/scripts/create-olm-bundle.sh ${{ inputs.tag }} ${{ steps.vars.outputs.previous_tag }} quay.io/${{ inputs.quay-org }}/keycloak-operator
```

Comment:
I'm not 100% sure `inputs.tag` contains the whole version string, incl. micro – i.e. 19.2.1. Can we rely on it? Shouldn't we rather use `inputs.version`?

Reply:
@vmuzikar The `inputs.tag` is also used for the checkout Keycloak branch, so it's aligned with it. And as we take the previous tag from the KC repository, I guess it's not a problem here.

Reply:
I've just noticed even for the original `operator.yml`, these properties are defined as:
```
version: ${{ github.event.inputs.**version** }}
tag: ${{ github.event.inputs.**version** }}
```

Reply:
Ok, we will have to try it anyway to see how it works. :) We just need to make sure we use `x.y.z` format.

Reply:
I've been thinking about it a little bit more, and it makes sense to use only one of them; preferred 'version'.